sim/druid/druid.go

```diff
@@ -111,6 +111,9 @@ const (
 	DruidSpellWildMushroom
 	DruidSpellWildMushroomDetonate
 	DruidSpellWrath
+	DruidSpellMightOfUrsoc
+	DruidSpellSavageDefense
+	DruidSpellFrenziedRegeneration
```

Comment:
We've been sorting these alphabetically, so it would be nice to stick to that convention when adding new flags.